cmd/juju/application/refresh.go

```diff
@@ -277,9 +280,11 @@ func (c *refreshCommand) SetFlags(f *gnuflag.FlagSet) {
 	f.BoolVar(&c.Force, "force", false, "Allow a charm to be refreshed which bypasses LXD profile allow list")
 	f.BoolVar(&c.ForceUnits, "force-units", false, "Refresh all units immediately, even if in error state")
 	f.StringVar(&c.channelStr, "channel", "", "Channel to use when getting the charm from Charmhub")
-	f.BoolVar(&c.ForceBase, "force-series", false, "Refresh even if series of deployed applications are not supported by the new charm")
+	f.BoolVar(&c.ForceBase, "force-series", false, "Refresh even if the series of the deployed application is not supported by the new charm")
+	f.BoolVar(&c.ForceBase, "force-base", false, "Refresh even if the base of the deployed application is not supported by the new charm")
```

Comment:
in help, these lines show up as:
```
--force-base, --force-series  (= false)
    Refresh even if the base of the deployed application is not supported by the new charm
```
Can we remove to deprecate `force-series`, appending ". DEPRECATED: use --series" to the description.

Reply:
I assume you mean DEPRECATED: use --force-base, but I have done so.